security-center/snippets/v2/addFindingSecurityMarks.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2024 Google LLC
```

Comment:
question: These samples appear identical to the v1 snippets. Aside from the import statement pulling in v2, are there other required changes?

Reply:
@vijaykanthm can you respond to this ?

Reply:
@grayside No, I don't think there are any changes.